src/backend/marsha/core/migrations/0011_auto_20190417_0925.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.0 on 2019-04-17 09:25
```

Comment:
in commit message: it is called "field" in Django models. A property would be just an attribute on the class but would not be understood to be linked to a database column.

Reply:
:ok_hand: